### PR TITLE
ENG-5180 feat(1ui): add network stats component to storybook

### PR DESCRIPTION
**Pull request by Vitalsine85**
## Affected Packages

Apps

- [ ] data populator
- [ ] portal
- [ ] template

Packages

- [x] 1ui
- [ ] api
- [ ] graphql
- [ ] protocol
- [ ] sdk

Tools

- [ ] tools

## Overview

Adds the NetworkStats component to Storybook for use in the Launchpad.

## Screen Captures

If applicable, add screenshots or screen captures of your changes.

## Declaration

- [x] I hereby declare that I have abided by the rules and regulations as outlined in the [CONTRIBUTING.md](https://github.com/0xIntuition/intuition-ts/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: packages/1ui/src/components/NetworkStats/NetworkStats.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    tvl: 13.34,
+    atomsCount: 4200,
+    triplesCount: 4200,
+    signalsCount: 4200,
+    usersCount: 4200,
+  },
+}
+
+const SmartNetworkStats = () => {
+  const { data: systemStats } = useGetStatsQuery(
```
Comment (jonathanprozzi) [inline packages/1ui/src/components/NetworkStats/NetworkStats.stories.tsx:36]: Ahh if you were having issues with this query in the component we may need to do some additional config at the decorator level to support the new schema! It'll work fine in launchpad -- will look into this if you were still seeing it.

File: packages/1ui/src/components/NetworkStats/NetworkStats.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { cn } from 'styles'
+import { formatNumber } from 'utils'
```
Comment (jonathanprozzi) [inline packages/1ui/src/components/NetworkStats/NetworkStats.tsx:2]: Not that it's top priority at the moment but I think that this highlights the need for us to make a `utils` package where we combine all of these + Portal into one place.

**Review by jonathanprozzi — APPROVED** [review]
Nice job! Some comments for us to think about moving forward

File: packages/1ui/src/components/NetworkStats/NetworkStats.tsx (hunk 2 above)
Comment (Vitalsine85) [reply packages/1ui/src/components/NetworkStats/NetworkStats.tsx:2]: Agreed

**Pull request merged**